app-android/src/main/res/drawable-night-v31/splash_icon.xml

```diff
@@ -0,0 +1,114 @@
+<animated-vector
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:aapt="http://schemas.android.com/aapt">
+    <aapt:attr name="android:drawable">
+        <vector
+            android:name="vector"
+            android:width="108dp"
+            android:height="108dp"
+            android:viewportWidth="216"
+            android:viewportHeight="216">
+            <group
+                android:name="group"
+                android:translateX="52"
+                android:translateY="52"
+                >
```

Comment:
I had to manually modify the material provided to me because it was clipped when used as is.